spring-cloud-dataflow-server-kubernetes-docs/pom.xml

```diff
@@ -89,6 +99,27 @@
 									</attributes>
 								</configuration>
 							</execution>
+							<execution>
+								<id>generate-html5</id>
+								<phase>prepare-package</phase>
+								<goals>
+									<goal>process-asciidoc</goal>
+								</goals>
+								<configuration>
+									<backend>html5</backend>
+									<attributes>
+										<docinfo>true</docinfo>
```

Comment:
Some of those can be removed, as they were moved up at line 71+.
Will take care of it